### ISSUE TITLE
PSR2.Classes.PropertyDeclaration rule

### ISSUE THREAD

**Issue by ilazaridis** [squizlabs/PHP_CodeSniffer#2769]
Hello,

I am using phpcs v3.5.3.

The problem is that I already have in my ruleset the following rule:
`<exclude ref="PSR2.Classes.PropertyDeclaration"/>`

But I still see errors like this: 
`1 | WARNING | [ ] Property name "$_blabla" should not be prefixed with an underscore to indicate visibility`

Am I excluding the wrong rule ?

**Comment by gsherwood** [comment]
First, the `exclude` tag uses `name` instead of `ref`, so the current rule wont work.

But also check that you've got the exclude tag inside the PSR2 rule tag, like this:
```xml
<rule ref="PSR2">
    <exclude name="PSR2.Classes.PropertyDeclaration"/>
</rule>
```
If you have the tag outside, it wont work:
```xml
<rule ref="PSR2" />
<exclude name="PSR2.Classes.PropertyDeclaration"/>
```
Does that help resolve the problem?

**Comment by ilazaridis** [comment]
`ref` was the problem. Replacing with `name` solved the issue. Thanks.

By the way I use this inside the PSR12 rule tag, which is also using it as I checked and it's working fine.

**Comment by gsherwood** [comment]
> Thank you @gsherwood

You're welcome
